### PR TITLE
refactor: move markdown to template

### PR DESCRIPTION
**Pull request by mindspank**
* Removes the image renderer that was not used
* Moves the markdown renderer to the new template syntax
* Adjusts default custom dashboard yaml to new syntax
* Adds css support to the markdown component

### CODE REVIEW

**Review by begelundmuller — APPROVED** [review]
Backend changes look good!

File: web-common/src/features/templates/markdown/Markdown.svelte [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+  import DOMPurify from "dompurify";
+  import { marked } from "marked";
+  import { MarkdownProperties } from "@rilldata/web-common/features/templates/types";
+  import { V1ComponentSpecRendererProperties } from "@rilldata/web-common/runtime-client";
+
+  export let rendererProperties: V1ComponentSpecRendererProperties;
+
+  $: kpiProperties = rendererProperties as MarkdownProperties;
+  $: css = kpiProperties.css || {};
```
Comment (djbarnwal) [inline web-common/src/features/templates/markdown/Markdown.svelte:10]: `kpiProperties` should be renamed to `markdownProperties`

**Review by djbarnwal — APPROVED** [review]
Left a small nit comment. Otherwise looks good. Feel free to merge after addressing the nit

File: web-common/src/features/templates/markdown/Markdown.svelte (hunk 1 above)
Comment (mindspank) [reply web-common/src/features/templates/markdown/Markdown.svelte]: Opps! Fixed!

**Pull request merged**